Textor/Controller/SettingsViewController.swift

```diff
@@ -25,6 +28,10 @@ class SettingsViewController: UITableViewController {
 		
 		NotificationCenter.default.addObserver(self, selector: #selector(didChangeTheme), name: .themeChanged, object: nil)
 		
+		defaultSeparatorColor = tableView.separatorColor
+		let components = defaultSeparatorColor.cgColor.components!
+		invertedSeparatorColor = UIColor(red: 1 - components[0], green: 1 - components[1], blue: 1 - components[2], alpha: components[3])
```

Comment:
> It's probably best to add this as an extension on UIColor. 

Unfortunately, this wouldn't really work all that well since it requires a valid reference to a `UITableView` to get the color from. That's why I stuck it in `viewDidLoad()`.

> Furthermore, I don't like the unsafe unwrapping `defaultSeparatorColor.cgColor.components` here.

Normally, I'd agree with you; however, I'm pretty sure that this property is either improperly annotated for Swift, or returns `nil` if the color is not a valid `CGColor`. `CGColor.components` is equivalent to the C method  `CGColorGetComponents`, which returns a `CGFloat *`. At first, I assumed that `nil` would mean there were no components, but apparently there are *always* components:

> The size of the array is one more than the number of components of the color space for the color.

So, the unwrapping operation is either safe or it uncovers a bug in how we're using the color, such as the separator color not being a valid `CGColor` (!)